zokrates_js/src/lib.rs

```diff
@@ -32,12 +35,6 @@ pub struct ComputationResult {
     output: String,
 }
 
-impl ResolverResult {
```

Comment:
Was this not being used?

Reply:
Yeah but I feel like this impl is unnecessary